src/jit/codegen.h

```diff
@@ -443,26 +443,26 @@ protected :
 
     void                genPrologPadForReJit();
 
-    void                genEmitCall(int                                                 callType,
-                                    CORINFO_METHOD_HANDLE                               methHnd,
-                                    INDEBUG_LDISASM_COMMA(CORINFO_SIG_INFO*             sigInfo)
-                                    void*                                               addr
-                                    X86_ARG(ssize_t                                     argSize),
-                                    emitAttr                                            retSize
-                                    FEATURE_UNIX_AMD64_STRUCT_PASSING_ONLY_ARG(emitAttr secondRetSize),
-                                    IL_OFFSETX                                          ilOffset,
-                                    regNumber                                           base   = REG_NA,
-                                    bool                                                isJump = false,
-                                    bool                                                isNoGC = false);
+    void                genEmitCall(int                                           callType,
+                                    CORINFO_METHOD_HANDLE                         methHnd,
+                                    INDEBUG_LDISASM_COMMA(CORINFO_SIG_INFO*       sigInfo)
+                                    void*                                         addr
+                                    X86_ARG(ssize_t                               argSize),
+                                    emitAttr                                      retSize
+                                    MULTIREG_HAS_SECOND_GC_RET_ONLY_ARG(emitAttr  secondRetSize),
```

Comment:
This name predates my changes.  
We also have 'retSize' which is the main return type.
I had to enable the second return type changes that previously were only available on UNIX_AMD64.

I didn't really like the conditional parameter stuff, but needed to enable it to allow ARM64 to report both return values.